plotly/tools.py

```diff
@@ -1475,6 +1477,678 @@ class FigureFactory(object):
+        ```
+
+        Example 2: Index by Column with Numerical Entries
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        # Make data for chart
+        df = [dict(Task="Job A", Start='2009-01-01',
+                   Finish='2009-02-30', Complete=10),
+              dict(Task="Job B", Start='2009-03-05',
+                   Finish='2009-04-15', Complete=60),
+              dict(Task="Job C", Start='2009-02-20',
+                   Finish='2009-05-30', Complete=95)]
+
+        # Create a figure with Plotly colorscale
+        fig = FF.create_gantt(df, colors='Blues', index_col='Complete',
+                              show_colorbar=True, bar_width=0.5,
+                              showgrid_x=True, showgrid_y=True)
+
+        # Plot the data
+        py.iplot(fig, filename='Numerical Entries', world_readable=True)
+        ```
+
+        Example 3: Index by Column with String Entries
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        # Make data for chart
+        df = [dict(Task="Job A", Start='2009-01-01',
+                   Finish='2009-02-30', Resource='Apple'),
+              dict(Task="Job B", Start='2009-03-05',
+                   Finish='2009-04-15', Resource='Grape'),
+              dict(Task="Job C", Start='2009-02-20',
+                   Finish='2009-05-30', Resource='Banana')]
+
+        # Create a figure with Plotly colorscale
+        fig = FF.create_gantt(df, colors=['rgb(200, 50, 25)',
+                                          (1, 0, 1),
+                                          '#6c4774'],
+                              index_col='Resource',
+                              reverse_colors=True,
+                              show_colorbar=True)
+
+        # Plot the data
+        py.iplot(fig, filename='String Entries', world_readable=True)
+        ```
+
+        Example 4: Use a dictionary for colors
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        # Make data for chart
+        df = [dict(Task="Job A", Start='2009-01-01',
+                   Finish='2009-02-30', Resource='Apple'),
+              dict(Task="Job B", Start='2009-03-05',
+                   Finish='2009-04-15', Resource='Grape'),
+              dict(Task="Job C", Start='2009-02-20',
+                   Finish='2009-05-30', Resource='Banana')]
+
+        # Make a dictionary of colors
+        colors = {'Apple': 'rgb(255, 0, 0)',
+                  'Grape': 'rgb(170, 14, 200)',
+                  'Banana': (1, 1, 0.2)}
+
+        # Create a figure with Plotly colorscale
+        fig = FF.create_gantt(df, colors=colors,
+                              index_col='Resource',
+                              show_colorbar=True)
+
+        # Plot the data
+        py.iplot(fig, filename='dictioanry colors', world_readable=True)
+        ```
```

Comment:
Can you add an example of using a `dataframe` here? You've only got lists of dicts in the :dolls: ^^.